src/com/apeshko/javacore/task_07/ProdRunner.java

```diff
@@ -0,0 +1,39 @@
+package com.apeshko.javacore.task_07;
+
+import com.apeshko.javacore.task_06.ClassesLoader;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class ProdRunner {
+    public static Annotation getAnnotation(Annotation[] annotations, String annotationName) {
+        for (Annotation annotation : annotations) {
+            if (annotation.annotationType().getName().contains(annotationName)) {
+                return annotation;
+            }
+        }
+
+        return null;
+    }
+
+    public static void main(String... args) {
+        ClassLoader classLoader = com.apeshko.javacore.task_06.Application.class.getClassLoader();
+
+        Iterable<Class> classes = new ClassesLoader().getClasses(classLoader, "com.apeshko.javacore.task_05");
+
+        for (Class clazz : classes) {
```

Comment:
try to move this logic to service class, with method like 'handle(), process()... etc'